autoload/coverage/gcov/parsing.vim

```diff
@@ -35,7 +35,8 @@ let s:plugin = maktaba#plugin#Get('coverage')
 " geninfo's man page, or view the man page at:
 " http://ltp.sourceforge.net/coverage/lcov/geninfo.1.php
 function! s:TryParseLine(line) abort
-  let [l:prefix, l:raw_info] = split(a:line, ':', 1)
+  let [l:prefix; l:raw_info] = split(a:line, ':', 1)
```

Comment:
Is the semicolon here instead of comma a typo?

Reply:
No, it's a way to collect multiple remaining list elements into a list variable when doing list unpacks.

```
When the number of variables does not match the number of items in the list
this produces an error.  To handle any extra items from the list append ";"
and a variable name: 

        :let [var1, var2; rest] = mylist

This works like: 
        :let var1 = mylist[0]
        :let var2 = mylist[1]
        :let rest = mylist[2:]
```
- https://vimhelp.org/eval.txt.html#Lists

Reply:
Oh nice, #TIL.